src/GraphType.jl

```diff
@@ -301,7 +301,7 @@ mutable struct Graph
                 adjdict[p0][p1] = j1
 
                 bm = trues(spp[p1], spp[p0])
-                bmt = transpose(bm)
+                bmt = permutedims(bm)
```

Comment:
This is much less efficient than `copy(bm')`.

Reply:
Why? It should be doing the same thing.

Reply:
It calls the generic `permutedims!` method in "base/multidimensional.jl" rather than the specialized `transpose!` method in "stdlib/LinearAlgebra/src/bitarray.jl"

Reply:
The solution seems to be to add a specialized method for this operation then, no?

Reply:
Oh sorry. I thought they would end up being the same thing. I will change it back then. I didn't feel like LinearAlgebra stuff would be needed here.

Reply:
> The solution seems to be to add a specialized method for this operation then, no?

That can be done, I guess. We would need to bring back the transpose code from `LinearAlgebra` into `Base`. It would also mean that we would need to specialize `permitedims!`  for `BitArray`s with the `perm=(2,1)` argument value. (At this point, it seems weird not to call it `transpose` though...)

Reply:
Ok, maybe just revert this for now and open an issue to figure out the best approach to having a fallback to address the performance.

Reply:
What is the absolute numbers in performance here though? It is a bit unfortunate to have Pkg3 depend on the Linear Algebra stdlib since presumably people will want to disable that one while still be able to use the package manager.